src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -181,48 +185,63 @@ export class TelemetryManagementSection extends Component<Props, State> {
-                  defaultMessage="Privacy Statement"
-                />
-              </EuiLink>
-            ),
-          }}
-        />
-      </p>
-      <p>
-        <FormattedMessage
-          id="telemetry.seeExamplesOfWhatWeCollect"
-          defaultMessage="See examples of the {clusterData} and {endpointSecurityData} that we collect."
-          values={{
-            clusterData: (
-              <EuiLink onClick={this.toggleExample}>
-                <FormattedMessage id="telemetry.clusterData" defaultMessage="cluster data" />
-              </EuiLink>
-            ),
-            endpointSecurityData: (
-              <EuiLink onClick={this.toggleSecurityExample}>
-                <FormattedMessage
-                  id="telemetry.securityData"
-                  defaultMessage="endpoint security data"
-                />
-              </EuiLink>
-            ),
-          }}
-        />
-      </p>
-    </Fragment>
-  );
+  renderDescription = () => {
+    const { isSecurityExampleEnabled } = this.props;
+    const securityExampleEnabled = isSecurityExampleEnabled();
+    const clusterDataLink = (
+      <EuiLink onClick={this.toggleExample} data-test-id="cluster_data_example">
+        <FormattedMessage id="telemetry.clusterData" defaultMessage="cluster data" />
+      </EuiLink>
+    );
+
+    const endpointSecurityDataLink = (
+      <EuiLink onClick={this.toggleSecurityExample} data-test-id="endpoint_security_example">
+        <FormattedMessage id="telemetry.securityData" defaultMessage="endpoint security data" />
+      </EuiLink>
+    );
+
+    return (
+      <Fragment>
+        <p>
+          <FormattedMessage
+            id="telemetry.telemetryConfigAndLinkDescription"
+            defaultMessage="Enabling data usage collection helps us manage and improve our products and services.
+            See our {privacyStatementLink} for more details."
+            values={{
+              privacyStatementLink: (
+                <EuiLink href={PRIVACY_STATEMENT_URL} target="_blank">
+                  <FormattedMessage
+                    id="telemetry.readOurUsageDataPrivacyStatementLinkText"
+                    defaultMessage="Privacy Statement"
+                  />
+                </EuiLink>
+              ),
+            }}
+          />
+        </p>
+        <p>
+          {securityExampleEnabled ? (
+            <FormattedMessage
+              id="telemetry.seeExampleOfClusterDataAndEndpointSecuity"
+              defaultMessage="See examples of the {clusterData} and {endpointSecurityData} that we collect."
+              values={{
+                clusterData: clusterDataLink,
+                endpointSecurityData: endpointSecurityDataLink,
+              }}
+            />
+          ) : (
+            <FormattedMessage
+              id="telemetry.seeExampleOfClusterData"
+              defaultMessage="See example of the {clusterData} that we collect."
```

Comment:
Missed pluralization of 'example', I think? "See an example of...." or "See examples of...." ?

Reply:
Its just 1 example of the clusterData. examples in the case we have 2 examples 1 for clusterData and the other for security solutions. I might be wrong but that was my reasning behind it

Reply:
Fixed. thanks!